lib/engines/content_block_manager/features/edit_object.feature

```diff
@@ -81,16 +82,31 @@ Feature: Edit a content object
     | my email address | xxxxx           | Ministry of Example |
     Then I should see a message that the "email_address" field is an invalid "email"
 
+  Scenario: GDS editor sees validation errors for unconfirmed answers
```

Comment:
discussed here https://gds.slack.com/archives/C0776B04EJU/p1733832855462279?thread_ts=1733831595.019639&cid=C0776B04EJU moving these type of error cases to the workflow_test, but again this PR is taking ages so might do that in a refactoring PR!